xml/System.Security.Cryptography/SHA256Managed.xml

```diff
@@ -201,63 +201,62 @@ This method is called by the public <xref:System.Security.Cryptography.HashAlgor
         <Parameter Name="start" Type="System.Int32" Index="1" FrameworkAlternate="xamarinios-10.8" />
-        <Parameter Name="ibStart" Type="System.Int32" Index="2" FrameworkAlternate="netcore-2.0;netcore-2.1;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netstandard-2.0;xamarinandroid-7.1" />
+        <Parameter Name="cbSize" Type="System.Int32" Index="2" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netframework-1.1;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;netstandard-2.0;xamarinandroid-7.1;xamarinmac-3.0" />
         <Parameter Name="length" Type="System.Int32" Index="2" FrameworkAlternate="xamarinios-10.8" />
-        <Parameter Name="cbSize" Type="System.Int32" Index="3" />
       </Parameters>
       <Docs>
         <param name="array">To be added.</param>
+        <param name="rgb">To be added.</param>
         <param name="data">To be added.</param>
-        <param name="rgb">The input data.</param>
+        <param name="ibStart">To be added.</param>
         <param name="start">To be added.</param>
-        <param name="ibStart">The offset into the byte array from which to begin using data.</param>
+        <param name="cbSize">To be added.</param>
         <param name="length">To be added.</param>
-        <param name="cbSize">The number of bytes in the array to use as data.</param>
-        <summary>When overridden in a derived class, routes data written to the object into the <see cref="T:System.Security.Cryptography.SHA256" /> hash algorithm for computing the hash.</summary>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- This method is not called by application code.  
-  
- This abstract method performs the hash computation. Every write to the cryptographic stream object passes the data through this method. For each block of data, this method updates the state of the hash object so a correct hash value is returned at the end of the data stream.  
-  
- ]]></format>
-        </remarks>
-        <altmember cref="T:System.Security.Cryptography.HashAlgorithm" />
-        <related type="Article" href="~/docs/standard/security/cryptographic-services.md">Cryptographic Services</related>
+        <summary>To be added.</summary>
+        <remarks>To be added.</remarks>
```

Comment:
This is, a one time side-effect of recovering from gh issue https://github.com/mono/api-doc-tools/issues/291 … because the parameters had incorrect indexes, mdoc is unable to match the `<Member>` element with the member definition (it does so based on name, number of parameters, etc). So in this instance, it ends up creating a new XML node and discarding the old one :( 

It's unfortunate, but this _kind_ of bug is very rare (an issue with the enumerator that matches xml node to reflection definition), so it shouldn't happen again after this run (which fixes the parameter list going forward)

Reply:
Ok @joelmartinez. I'm trying to see where those happen to recover the content.